src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -155,6 +157,33 @@ export CONDUCTR_HAPROXY_CONFIG_OVERRIDE="$CONFIG_DIR/haproxy-override.cfg"
 The `CONFIG_DIR` refers to the directory where the configuration override will be expanded. `CONDUCTR_HAPROXY_CONFIG_OVERRIDE` is the environment variable which tells ConductR where to find the custom HAProxy configuration template.
 
 
+### Specifying custom HAProxy reload script
+
+ConductR HAProxy makes use of the reload script to reload HAProxy whenever new HAProxy configuration is generated. When ConductR HAProxy starts, the reload script is copied from the path specified by `HAPROXY_RELOAD_SCRIPT_SOURCE` environment variable to the path specified by `HAPROXY_RELOAD_SCRIPT_LOCATION` environment variable.
+
+Create a file called `runtime-config.sh` within the proxy configuration directory, e.g.
```

Comment:
Introduce first what `runtime-config.sh` is for (As a reader I don't know why I'm creating a file called `runtime-config.sh`). It could be a one liner, something like 'conductr HAProxy sources a `runtime-config.sh` file to set these environment variables (and make sire that the proxy script can be copied over'